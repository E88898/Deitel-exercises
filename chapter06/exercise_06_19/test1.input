1 1
4 5
12 7
