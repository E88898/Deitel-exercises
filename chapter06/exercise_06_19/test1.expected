Enter 2 sides of right triangle: Hypotenuse: 1.4
Enter 2 sides of right triangle: Hypotenuse: 6.4
Enter 2 sides of right triangle: Hypotenuse: 14
